src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -138,7 +138,7 @@
                       :sort-value (if checked? 1 2)})
            :commands {:td [:td.commands
                            [view-button (:id item)]
-                           [roles/show-when roles/+admin-write-roles+
+                           [roles/show-when-correct-access roles/+admin-write-roles+ (get-in item [:organization :organization/id])
```

Comment:
Here we could express the `or` relationship somehow. So either (super) `owner` or an owner of this particular organization. Now it's a bit ambiguous. The "framework" we have with `show-when` doesn't seem to be enough.